src/main/java/com/example/MyBatisDemo/model/TeacherEntity.java

```diff
@@ -0,0 +1,20 @@
+package com.example.MyBatisDemo.model;
+
+import lombok.*;
+
+import java.util.List;
+
+@Data
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class TeacherEntity {
+
+    private Long id;
+    private String name;
+    private int age;
+    private String email;
+    private String City;
+    private SubjectEntity subject;
```

Comment:
A teacher can teach multiple subjects!! 😏

Reply:
updated in next commits